lengow.php

```diff
@@ -45,7 +45,7 @@ public function __construct()
     {
         $this->name = 'lengow';
         $this->tab = 'export';
-        $this->version = '3.5.3';
+        $this->version = '3.5.4';
```

Comment:
@michaelmaslengow Should this version be related to the project version ? In other words should I find a way to automaticly change this value ? Can it be prefixed with `v` like `v3.5.4` ?

Reply:
@A-Oudjat  Yes, this is the version of the plugin defined for Prestashop to detect.
No, it cannot be prefixed.

Reply:
@A-Oudjat this is the module version. it's required for module validator